core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -488,6 +488,17 @@ package object config {
       .booleanConf
       .createWithDefault(false)
 
+  private[spark] val STORAGE_REMOTE_SHUFFLE_MAX_DISK =
```

Comment:
this needs to be added to documentation configuration.md

Reply:
actually I don't see any of these in docs?  spark.storage.decommission.shuffleBlocks.enabled. sorry if I already asked this and we meant to , but these are not internal configs

Reply:
So the theory behind that is that these are mostly like provider level configuration, e.g. we wouldn't expect an end user to set this but someone like Cloudera or Amazon would configure these for their deployments.